doc/create_ramp_event.rst

```diff
@@ -53,7 +53,32 @@ This config file should look like::
       more precisely the :ref:`conda workers <conda_env_worker>` to have more
       information.
 
-Finally, you can easily deploy the event (adding both problem and event to the
+
+Before you continue make sure that:
+
+    1.  your private and public data are available.
+
+        For example, your data might be in:
+
+        - public data: `ramp_deployment/ramp-kits/<problem_name>/data`, and
+        - private data: `ramp_deployment/ramp-data/<problem_name>/data`.
+
+        Note that:
+
+        - test and train can be directories or files, depending on the
+          challenge
```

Comment:
You might not need this part anymore, now that we aren't using the directory structure example.